scripts/zones/Rolanberry_Fields_[S]/IDs.lua

```diff
@@ -37,6 +39,11 @@ zones[xi.zone.ROLANBERRY_FIELDS_S] =
             [17150279] = 17150280, -- -484.535 -23.756 -467.462
         },
 
+        ERLE_PH =
+        {
+            [erleID - 6] = erleID, -- -298.487 3.637 54.586
```

Comment:
I assume you've tried, but there isn't better way to do this because of the order of operations when the table get created, yes?

Reply:
Correct best I can tell there not a better way to do it that keeps the dynamic use of `GetFirstID`